blockchain/indexers/flatutreexoproofindex.go

```diff
@@ -460,11 +550,29 @@ func (idx *FlatUtreexoProofIndex) MakeMultiBlockProof(currentHeight, proveHeight
 		panic(err)
 	}
 
-	delHashes := make([]accumulator.Hash, 0, len(delsToProve))
+	delHashes := make([]utreexo.Hash, 0, len(delsToProve))
 	for _, del := range delsToProve {
 		delHashes = append(delHashes, del.LeafHash())
 	}
 
+	var stump utreexo.Stump
```

Comment:
This bit of code is just for sanity checking. Can remove this as it adds unnecessary performance hit